cdap-api/src/main/java/io/cdap/cdap/api/service/Service.java

```diff
@@ -38,7 +38,8 @@
   void configure(T configurer);
 
   /**
-   * Initialize the {@code Service}.
+   * Initialize the {@code Service}. This method will be called before the service starts up. It will be called
```

Comment:
This method will be called before each service instance starts up. If there is more than one instance of the service, it will be called more than once.